paasta_tools/kubernetes/bin/paasta_secrets_sync.py

```diff
@@ -38,10 +41,23 @@
 from paasta_tools.paasta_service_config_loader import PaastaServiceConfigLoader
 from paasta_tools.secret_tools import get_secret_provider
 from paasta_tools.utils import DEFAULT_SOA_DIR
+from paasta_tools.utils import get_service_instance_list
+from paasta_tools.utils import INSTANCE_TYPES
 from paasta_tools.utils import load_system_paasta_config
 
 log = logging.getLogger(__name__)
 
+INSTANCE_TYPE_TO_K8S_NAMESPACE = {
```

Comment:
Please move `INSTANCE_TYPE_TO_K8S_NAMESPACE` into `paasta_tools.utils next` next to `INSTANCE_TYPES`.

Reply:
didn't even notice that `INSTANCE_TYPES` was from there, definitely makes more sense there.